.github/workflows/test_functional.yml

```diff
@@ -78,8 +88,6 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-        with:
-          fetch-depth: 2  # required by codecov
```

Comment:
not needed since switching to the Codecov GH Action uploader instead of the bash uploader